mlstream/datautils.py

```diff
@@ -75,7 +75,8 @@ def load_discharge(data_root: Path, basins: List = None, file_format: str = 'nc'
     found_basins = []
     for f in files:
         q_nc = nc.Dataset(f, 'r')
-        file_basins = q_nc['station_id'][:]
+        file_basins_og = np.array([f for f in q_nc['station_id'][:]])
+        file_basins = np.array([f[-7:] for f in file_basins_og])

```

Comment:
Can you add a comment to explain what's happening here? What's the difference between `file_basins_og` and `file_basins`?

Reply:
Comment added.